app/lib/tool/utils/http.dart

```diff
@@ -17,14 +17,14 @@ http.Client httpRetryClient({
   http.Client? innerClient,
   int? retries,
 }) {
-  return RetryClient(
```

Comment:
hmm, when do we use this?

We should probably never use `RetryClient` as I don't think it retries if the connection fails when reading the response, I could be wrong...

Anyways, this is orthogonal.

Reply:
We have 8 or 9 references already in pub-dev. While it doesn't retry on certain failures, it was still better than using nothing, so I wouldn't remove it unless we have a better solution.